workbench/planning/reporting.py

```diff
@@ -294,6 +344,13 @@ def _project_record(self, project, offers):
         )
         hours = sum(rec["offer"]["planned_hours"] for rec in offers) if offers else 0
 
+        absences = (
+            [
+                ({"name": str(user), "short_name": user.get_short_name()}, lst)
+                for user, lst in sorted(self._project_absences[project].items())
+            ],
```

Comment:
Bist Du 100% sicher, dass das ein Tuple sein soll? Vermutlich ist das Komma da überflüssig?

Reply:
Das ist nicht ganz elegant, aber ich wollte im Frontend bereits bestehende Komponenten für die Absenzen verwenden, daher die Lösung mit Tupel und nicht einheitlich als Dictionary.

Reply:
Das Tuple in der Liste ist schon OK, aber `absences = ([....],)` ist auch ein Tuple, wegen dem Komma nach der eckigen Klammer.

Reply:
Hah, ne da hab ich nicht aufgepasst.